test/monitor_test.py

```diff
@@ -44,6 +44,8 @@ def Driver(success):
             # Two new objects.
             ray.get(ray.put(1111))
             ray.get(ray.put(1111))
+            # Test is flaky without calls to sleep.
```

Comment:
What are the sleeps needed for? Can we get rid of them?

Reply:
The test sometimes fails without the wait in place.

Reply:
Why does it fail, does this mean the cleanup implementation is buggy?

Reply:
`ray.get` completes before information about the object's location is propagated to the GCS, so the count returned by `StateSummary()` immediately after a `ray.get` is incorrect.

To compare: If the object was remote, then `ray.get` would only complete after data propagates to the GCS: The associated `pull` for `ray.get` depends on information in the GCS.